packages/webapp/src/webview/state/reducers.ts

```diff
@@ -112,6 +115,8 @@ function cloneState(state: AppState): AppState {
  */
 function updateGuidedAnswerTreesReducer(newState: AppState, action: UpdateGuidedAnswerTrees): AppState {
     newState.guidedAnswerTreeSearchResult = action.payload;
+    newState.updatedGuidedAnswerTrees = [...newState.updatedGuidedAnswerTrees, ...action.payload.trees];
+    newState.currentOffset = newState.currentOffset + PAGE_SIZE;
```

Comment:
Is there any time we can have different values for `newState.updatedGuidedAnswerTrees.length` and `currentOffset` or if you follow my proposal from above, the question is can it be that 
`newState.guidedAnswerTreeSearchResult.trees` is different than `newState.currentOffset`?

Reply:
Changed, thanks!